.travis.yml

```diff
@@ -5,6 +5,9 @@ node_js:
 
 language: node_js
 
+before_install:
+  - export TZ=America/Sao_Paulo
+
```

Comment:
[@lucianopf @travoul da pra usar o `env:` pra fazer isso não? [examplo](https://docs.travis-ci.com/user/environment-variables#defining-public-variables-in-travisyml)